moto/elb/responses.py

```diff
@@ -261,7 +385,22 @@ def describe_instance_health(self):
           <CanonicalHostedZoneNameID>Z3ZONEID</CanonicalHostedZoneNameID>
           <Scheme>internet-facing</Scheme>
           <DNSName>tests.us-east-1.elb.amazonaws.com</DNSName>
-          <BackendServerDescriptions/>
+          <BackendServerDescriptions>
+          {% for backend in load_balancer.backends %}
+            <member>
+                {% if backend.instance_port %}
+                <InstancePort>{{ backend.instance_port }}</InstancePort>
+                {% endif %}
+                {% if backend.policy_names %}
+                    <PolicyNames>
+                        {% for policy in backend.policy_names %}
+                            <member>{{ policy }}</member>
+                        {% endfor %}
+                    </PolicyNames>
+                    {% endif %}
+            </member>
+          {% endfor %}
+          </BackendServerDescriptions>
```

Comment:
This is the portion I'm talking about, when I say that I'm not sure that I've correctly modeled the backend server description objects.